app/controllers/cars_controller.rb

```diff
@@ -21,6 +21,12 @@ def create
     end
   end
 
+  def destroy
+    Car.find(params[:id]).destroy
+    flash[:success] = "Your car removed!"
```

Comment:
What is `flash`? Is it some kind of logging library?

Reply:
`flash` is a way of doing notices to user. It's like a Toast message in Android. More info [here](https://api.rubyonrails.org/classes/ActionDispatch/Flash.html)